unfetter-discover/unfetter-ui
# Fix route refresh

fixes unfetter-discover/unfetter#868

## To Test
* Pull this branch
* Verify tests, lint, and builds pass
* Bring up the app and route to page that is not 'not found', partners, or home
* edit code, verify page reloads and does not route to the home
* refresh browser, verify page reloads and does not route to the home
 * this second step may effect user sign in @j987987 please verify
* also this lazy loads the stix pages, probably verified by running webpack analyzer

Additionally this improves the close icon and dialog size for threat dashboard.
